### PR TITLE
add dependents tool to cleanup section

### PR DESCRIPTION
**Pull request by Fuzzyma**
Someone in our discord requested to add the tool to the cleanup section of the docs: https://discord.com/channels/1227627367204257852/1227653370945474590/1317529015191339009

Not a huge writer myself but I hope this will do.

### CODE REVIEW

File: docs/guide/cleanup.md
Comment (aldy505) [inline docs/guide/cleanup.md:139]:
```suggestion
Next, use [fuzzymas tool](https://github.com/fuzzyma/e18e-tools) to find the dependents of the package. You can find documentation on how to use the tool in its README.
```

File: docs/guide/cleanup.md
Comment (aldy505) [inline docs/guide/cleanup.md:144]: Should we point out something like who to contact or what to do when the CouchDB URL is not reachable?

I was thinking of providing some docs on how to host the database itself (in which someone would have to privately request the data to someone), but I don't think that's relevant enough for now.

Comment (Fuzzyma) [reply docs/guide/cleanup.md:144]: I think if they really want to do that, they can join the discord and just ask :)

File: docs/guide/cleanup.md
Comment (43081j) [inline docs/guide/cleanup.md:135]: i think this should be:

> Tip: Finding dependents of a package

Since the previous 3 sections are directly connected (i.e. discover -> improve -> fix), but this one won't be.

if we name it `Tip:`, it opens the door for us adding other tips in future

Comment (Fuzzyma) [reply docs/guide/cleanup.md]: Replaced it. Its all orange text now :D.
I dont mind it. Wdyt?

Comment (43081j) [reply docs/guide/cleanup.md]: i'd be tempted to keep it as a section since we might add more of them, and these tip boxes aren't visible in the contents list (the sidebar)

Comment (Fuzzyma) [reply docs/guide/cleanup.md]: Oh you meant to name the whole section "Tip: blabla"?

Comment (Fuzzyma) [reply docs/guide/cleanup.md]: done

**Review by 43081j — APPROVED** [review]
looks good

lets wait for one more review and we can merge 👍

**Review by bluwy — APPROVED** [review]

**Pull request merged**
